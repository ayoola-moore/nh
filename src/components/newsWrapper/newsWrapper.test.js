import React from "react";
import NewsWrapper from "./";

describe("Component", () => {
  const component = shallow(<NewsWrapper />);
  it("should render NewsWrapper properly", () => {
    expect(component).toMatchSnapshot();
  });
});
