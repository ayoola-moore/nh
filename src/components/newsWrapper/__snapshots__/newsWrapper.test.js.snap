// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should render NewsWrapper properly 1`] = `ShallowWrapper {}`;
